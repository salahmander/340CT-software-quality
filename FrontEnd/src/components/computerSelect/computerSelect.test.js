import React from 'react';
import { shallow } from 'enzyme';
import ReactDOM from 'react-dom';

import ComputerSelect from './computerSelect'

describe('item list ', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<ComputerSelect />, div);
        ReactDOM.unmountComponentAtNode(div);
      });

    it('should render correctly in "debug" mode and match the snapshot', () => {
        const component = shallow(<ComputerSelect debug />);
        expect(component).toMatchSnapshot();
      });
      
    });