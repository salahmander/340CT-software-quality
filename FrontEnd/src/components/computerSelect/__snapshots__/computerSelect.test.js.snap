// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`item list  should render correctly in "debug" mode and match the snapshot 1`] = `
<div
  className="computerSelect"
>
  <img />
  <div
    className="container"
  >
    <button
      className="linkButton"
      onClick={[Function]}
    >
      <h4>
        <b />
      </h4>
    </button>
    <p />
  </div>
</div>
`;
